uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request.controller.js

```diff
@@ -297,8 +297,8 @@ define([
      * Sets the "change_balance" attribute to the leave request
      * if a force balance change recalculation is needed
      */
-    function decideIfBalanceChangeNeedsAForceRecalculation () {
-      if (!vm.isRole('staff')) {
+    function decideIfBalanceChangeNeedsRecalculationOnBackend () {
+      if (isBalanceChangeRecalculationNeeded() && !vm.isRole('staff')) {
```

Comment:
Are we removing the `change_balance` property before sending it back to API(`toAPI`), because its not needed by backend.

Reply:
It **is** needed for backend, it is essential actually, since then backend decided if to skip recalculation in case of dates/times have not been changed.

Reply:
Ohh, my bad.